### ISSUE TITLE
Old rtcmhelpers file in pyrtcm module

### ISSUE THREAD

**Issue by foxittt** [semuconsulting/pyrtcm#7]
Hi
After installing pyrtcm module there are not sat2prn, cell2prn, id2prnsigmap functions in rtcmhelpers.py file.
But they are in github`s rtcmhelpers.py file.

**Comment by semuadmin** [comment]
Hi @foxittt ,

I'm not sure how you've installed pyrtcm, but I've just rechecked both the 0.2.7 Release assets on GitHub and the corresponding `pyrtcm-0.2.7.tar.gz` archive on PyPi and they both contain all the relevant methods.

I can only recommend you uninstall or delete ALL your existing pytrcm copies and reinstall using pip:

```shell
> python3 -m pip uninstall pyrtycm (_do this repeatedly until you get the WARNING: Skipping pyrtcm as it is not installed. message_)

> python3 -m pip install --upgrade pyrtcm
```

FYI I am working on a new version of pyrtcm which will automatically tag RTCM3 MSM message group attributes with their corresponding satellite/signal IDs. It's currently undergoing testing but should be included in the next release 0.2.8-beta sometime in the next few weeks.

Hope this helps

**Comment by semuadmin** [comment]
The latest provisional version is in this branch: https://github.com/semuconsulting/pyrtcm/tree/add-MSM-PRN/SIG-labelling - if you want to test it ahead of general release, be my guest.
